config/zephyr/chip-module/CMakeLists.txt

```diff
@@ -99,11 +99,17 @@ matter_add_gn_arg_bool  ("chip_mdns_minimal"                        CONFIG_WIFI)
 matter_add_gn_arg_bool  ("chip_mdns_minimal"                        CONFIG_NET_L2_ETHERNET)
 matter_add_gn_arg_bool  ("chip_mdns_platform"                       CONFIG_NET_L2_OPENTHREAD)
 matter_add_gn_arg_bool  ("chip_inet_config_enable_tcp_endpoint"     CONFIG_CHIP_BUILD_TESTS)
-matter_add_gn_arg_bool  ("chip_inet_config_enable_udp_endpoint"     CONFIG_NET_UDP)
```

Comment:
Why was this wrong? Couldn't you just set `CONFIG_NET_TCP=n` in your project if you didn't want it to be used by Matter?

Reply:
I made that because, if I remember well, CONFIG_NET_TCP=y breaks things when building the app. I can put it back to CONFIG_NET_TCP if you want

Reply:
It's up to you but it made sense to me even if some adjustments were needed to support Matter over TCP.